src/main/comments.js

```diff
@@ -976,6 +976,10 @@ function printTrailingComment(commentPath, print, options) {
   }
   const isBlock = util.isBlockComment(comment);
 
+  // Workaround for comments that are in the superClass definition ( extends ...)
```

Comment:
Is this workaround ok?
Feels dirty to me but there was no other way to get the parentNode of the parentNode

Reply:
@azz

Reply:
`commentPath.getParentNode(3) === "superClass"`?

or maybe it’s `2`.

Reply:
i fixed it, its a little bit longer but better understandable and maintainable :)
We should definitely work on some kind of documentation :) There are cool features build in, but its hard for people who are new to find them :)

Reply:
thanks for the quick feedback :)